packages/fxa-react/configs/storybooks.js

```diff
@@ -3,80 +3,79 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const { resolve } = require('path');
+const ModuleScopePlugin = require('react-dev-utils/ModuleScopePlugin');
 
+const allFxa = resolve(__dirname, '../../');
+const importPaths = [allFxa, resolve(__dirname, '../../../node_modules')];
 const additionalJSImports = {
   'fxa-react': resolve(__dirname, '../'),
   'fxa-shared': resolve(__dirname, '../../fxa-shared'),
 };
 
-const customizeWebpackConfig = ({ config, mode }) => {
-  // TypeScript loading
-  // https://bit.ly/fxa-storybook-ts
-  config.module.rules.push({
-    test: /\.(ts|tsx)$/,
-    loader: require.resolve('babel-loader'),
-    options: {
-      presets: [['react-app', { flow: false, typescript: true }]],
-    },
-  });
-
-  // SCSS loading
-  config.module.rules.push({
-    test: /\.s[ac]ss$/i,
-    use: ['style-loader', 'css-loader', 'sass-loader'],
-  });
-
-  // CSS loading
-  // Our various projects have differing CSS loading implementations;
-  // this aims to provide cross-project support
-  const cssRuleIndex = config.module.rules.findIndex(({ test }) =>
-    test.test('.css')
-  );
-  const cssLoader = {
-    test: /\.css$/i,
-    use: ['style-loader', 'css-loader'],
-  };
-  if (cssRuleIndex > -1) {
-    config.module.rules[cssRuleIndex] = cssLoader;
-  } else {
-    config.module.rules.push(cssLoader);
-  }
-
-  // SVG loading
-  // Our various projects have differing SVG loading implementations;
-  // this aims to provide cross-project support
-  let svgLoader;
-  const svgRule = config.module.rules.find(({ test }) => test.test('.svg'));
-  if (svgRule) {
-    svgLoader = {
-      loader: svgRule.loader,
-      options: svgRule.options || svgRule.query,
-    };
-  } else {
-    svgLoader = {
-      loader: require.resolve('file-loader'),
-      options: { name: 'static/media/[name].[hash:8].[ext]' },
-    };
-  }
-  config.module.rules.unshift({
-    test: /\.svg$/,
-    use: ['@svgr/webpack', svgLoader],
-  });
-
-  config.module.rules.push({
-    test: /\.(woff|woff2|eot|ttf)$/,
-    loader: require.resolve('file-loader'),
-  });
-
-  config.resolve.extensions.push('.ts', '.tsx', '.svg', '.scss', '.css');
-  config.module.rules = [{ oneOf: config.module.rules }];
-  config.resolve.alias = Object.assign(
-    config.resolve.alias,
-    additionalJSImports
-  );
-
-  return config;
-};
+const customizeWebpackConfig = ({ config }) => ({
+  ...config,
```

Comment:
Oh, and I don't think I *had* to totally rewrite how this function customizes the webpack config. But, it's where I ended up after a long & twisty journey of trying to understand what was going on with config from different packages. Lots of console logging and re-adding tweaks one-by-one until things worked.